src/components/Footer.tsx

```diff
@@ -1,11 +1,20 @@
+import { useEffect, useState } from 'react';
 import Link from 'next/link';
 import { Github } from 'lucide-react';
 import { Button } from './ui/button';
 import { useTheme } from 'next-themes';
 
 const RsSchoolIcon = () => {
+  const [mounted, setMounted] = useState(false);
   const { resolvedTheme } = useTheme();
-  const isLight = resolvedTheme === 'light';
+  const isLightTheme = resolvedTheme === 'light';
```

Comment:
I've revisit the footer component in UI-Kit migration, so you can revert it

Reply:
I found a bug with Logo in the footer not changing properly, when I go back and forth on the page. I found [the solution](https://stackoverflow.com/questions/71706064/react-18-hydration-failed-because-the-initial-ui-does-not-match-what-was-render), that uses state and useEffect to rerender it properly

Reply:
Got  it!